project/pimcore.py

```diff
@@ -0,0 +1,14 @@
+from .remote import RemoteProject
+
+
+class Pimcore5(RemoteProject):
+    major_version = 'v5.8.2'
+    remote = 'https://github.com/pimcore/pimcore.git'
```

Comment:
I think this is the issue.  If we're building off of pimcore/skeleton, then pimcore/skeleton is what should be listed as the remote.  That's all composer create-project does anyway.  That also then removes the need for the funky tmp_build directory in platformify.

Reply:
Good catch, using the skeleton simplifies a lot the process.